sdks/python/apache_beam/examples/snippets/transforms/aggregation/groupby_test.py

```diff
@@ -38,6 +38,10 @@
 from .groupby_simple_aggregate import simple_aggregate
 from .groupby_two_exprs import groupby_two_exprs
 
+#
+# Temporarily skip all tests in file
+__test__ = False
```

Comment:
do you plan to enable this later?

Reply:
I created a spinoff issue since I think fixing the tests (which were already broken before these changes) is probably out of scope

https://github.com/apache/beam/issues/30778

Reply:
Probably preferable to comment out the incorrect asserts rather than disable the tests altogether (which will at least confirm the examples are syntactically correct and run without errors).

Reply:
(Though if you're committing to fix them ASAP, we can get this in.)